source/fundamentals/connection/connection-options.txt

```diff
@@ -46,7 +46,7 @@ parameters of the connection URI to specify the behavior of the client.
      - Specifies the database that connections should authenticate against.
 
    * - **compressors**
-     - comma separated list of strings, e.g. "snappy,zlib,zstd"
+     - comma separated list of strings, e.g. "snappy,zlib"
```

Comment:
S: [Don't use "e.g."](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#e)

Fine if you consider this outside the scope of this ticket.

Reply:
I try not to consider all fixes around every fix I make since that's usually outside the scope, but happy to update as part of this PR since it's a quick fix. Is the suggestion to remove the part starting with ", e.g."?

Reply:
Also, addressing it in this PR won't fix versions 4.7 and later as this is only for v4.6 and earlier. I can ticket that part.

Reply:
Since there are multiple instances of this on the page, I'll just ticket to make it a more straightforward fix.

Reply:
https://jira.mongodb.org/browse/DOCSP-28396